ios/Podfile.lock

```diff
@@ -644,7 +644,7 @@ SPEC CHECKSUMS:
   EXScreenOrientation: d94f791c7ade3639352f34fcf5353f7c554b7d69
   EXWebBrowser: 76783ba5dcb8699237746ecf41a9643d428a4cc5
   FBLazyVector: e686045572151edef46010a6f819ade377dfeb4b
-  FBReactNativeSpec: b96f20d2bcabd43e9acdeff6210208026a13633a
+  FBReactNativeSpec: e4ad5d0b6f62309f75aeadea6a6fe1b665816402
```

Comment:
Interesting. Do you know why this line is here?

I'd be interested to know what happens if you do the following:

1. Remove this change from the commit (could do `git checkout origin/main -- ios/Podfile.lock` then `git commit --amend`)
2. Run `yarn`.

Does this change reappear?

Reply:
1. @chrisbobbe!, Did you remember I asked you that I'm facing some issue while running the app, So you suggested me to  run `yarn` and I did that and I got the app running successfully, after that I saw that the `Podfile.lock` got changed. And my simulator changed from iPhone 11 to iPhone 12. Actually I wanted to ask you regarding this but forgot. So this is, the whole story.
2. Surely, I'll run the command as you have suggested above. Will update the progress asap. Thanks 😊.

Reply:
@chrisbobbe!, I did what you suggested, actually before doing `yarn` I got this `FBReactNativeSpec: b96f20d2bcabd43e9acdeff6210208026a13633a`, but when I did `yarn` then again it updated to the current one
`FBReactNativeSpec: e4ad5d0b6f62309f75aeadea6a6fe1b665816402`. Now, what should we do?

Reply:
Interesting! I think this is probably the problem that facebook/react-native@bdfe2a517 is meant to fix. There's an issue where that hash changes with an absolute path, and absolute paths are likely to be different on different computers. (For example, you probably don't have any absolute paths that start with `/Users/chrisbobbe/`.)

That's a change in the `react-native` package, and it looks like it was released in RN v0.65. It's a pretty annoying bug. (@gnprice, I guess you haven't run into it because you do most day-to-day development on Android, and I've been doing the recent iOS release builds. 😛)

For now, until we upgrade to RN v0.65 or find another solution, maybe the thing to do is just remove this change from your commits, unfortunately.

Reply:
Okay @chrisbobbe!, but I've a question, when I'll run app it will not run, and ultimately I'll have to do `yarn` which in case, It will modify the `Podfile.lock` file. In this case, what to do?

Reply:
> when I'll run app it will not run

It would be helpful to see specific error output, but my guess is you can keep the change there to avoid any problems in local development, and then just leave the change out of what you push to GitHub for the PR.

Reply:
Yeah @chrisbobbe! that could be one of the possible solution that we can do, until we upgrade to RN v0.65. Okay, then I'll remove that line from `Podfile.lock` and push the changes. I guess that will work. Thanks

Reply:
@chrisbobbe!, I tried to do what you said!, the thing is when I run `git checkout origin/main -- ios/Podfile.lock`, if you remember, then I got only `Podfile.lock` as we wanted, other than that everything was same as I have in PR. But then you said, keep that change for temporary purpose for yourself, and put 
`FBReactNativeSpec: b96f20d2bcabd43e9acdeff6210208026a13633a` in place of 
`FBReactNativeSpec: e4ad5d0b6f62309f75aeadea6a6fe1b665816402`. So I did that, but now `Podfile.lock` is unchanged. As a result, I've no changed file, so do I need to directly push it or I need to do something else?. Actually never faced this situation, If you could guide me, It would be a great help. Thanks 🙂.

Reply:
I don't yet understand the problem you're seeing now. 🙂

> As a result, I've no changed file

Can you say more about this, please? Your local commit(s) _should_ have some changed files. Specifically, you should see the changes you want to make in the PR, when you use Greg's "secret" to using `git log -p` (details are in our docs). You won't want to see any changes to `ios/Podfile.lock`, in the commits you push to the PR branch on GitHub.

To edit commits before pushing them to GitHub, please see our doc on that: https://zulip.readthedocs.io/en/latest/git/fixing-commits.html

Reply:
Try directly pushing the branch. What I'm seeing on GitHub still has the change to `ios/Podfile.lock`, and I'd like to see the PR without that change. Does that answer your question?

I think you do not need to edit the commit message to mention that `ios/Podfile.lock` is unchanged. Most commits leave `ios/Podfile.lock` unchanged without mentioning that it's unchanged. 🙂 

If you have ideas for improving the commit message to give more context or justification for the changes that _are_ made in the commit, you can do that if you want, of course.